src/scripts/components/ShapesIE.ts

```diff
@@ -1,19 +1,23 @@
 import Component from './Component'
 
-interface ShapesData {
+interface ShapesIEData {
 	url: string
 }
 
 /**
- * Shapes component class
+ * ShapesIE component class (Internet Explorer fallback)
  *
  * - injects SVG sprite into body
  */
-export default class Shapes extends Component<ShapesData> {
+export default class ShapesIE extends Component<ShapesIEData> {
 
-	static componentName = 'Shapes'
+	static componentName = 'ShapesIE'
 
 	init() {
+		if (window.navigator.userAgent.indexOf('MSIE ') < 0 && !navigator.userAgent.match(/Trident.*rv\:11\./)) {
+			return
+		}
+
 		document.implementation.hasFeature('http://www.w3.org/TR/SVG11/feature#BasicStructure', '1.1') && this.injectSprite()
```

Comment:
The [`hasFeature method is deprecated`](https://developer.mozilla.org/en-US/docs/Web/API/DOMImplementation/hasFeature) but do we really need it anyway? Are we even targeting browsers that don't support this stuff anymore? I think it will be safe to just call `injectSprite` from here directly.

Reply:
:+1: `hasFeature` condition removed